composer.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "fill-me-in/fill-me-in",
+    "license": "MIT",
+    "description": "fill-me-in",
+    "type": "project",
+    "require": {
+        "PHP": "^7.1",
+        "keboola/csv": "^1.2",
+        "keboola/php-component": "^2.0"
+    },
+    "require-dev": {
+        "jakub-onderka/php-parallel-lint": "^1.0",
+        "keboola/php-temp": "^1.0",
+        "phpstan/phpstan-shim": "^0.9.2",
+        "phpunit/phpunit": "^7.0",
+        "slevomat/coding-standard": "4.4.6",
+        "squizlabs/php_codesniffer": "~3.2",
+        "symfony/process": "^4.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "MyComponent\\": "src/"
```

Comment:
Přejmenovat namespace + upravit tady a v `autoload-dev`

Reply:
ja nevim, me ten namespace vubec nevadi :)

Reply:
Ordnung muss sein! Ale jsem otevřený tomu to prodiskutovat s ostatními...

Reply:
vymekl jsem :)

Reply:
delal jsem to refactoringem a potom rucne, protoze pres ref se to nepovedlo a je to na 5 mistech jenom